src/guide/profiles.md

```diff
@@ -25,6 +27,7 @@ All profiles are prepended by an `@` and in snake case, while all mutators are i
         "text": "infection.log"
     },
     "mutators": {
+        "global-ignore": [],
```

Comment:
Would you mind adding examples here?

Also, let's update https://infection.github.io/guide/how-to.html#How-to-disable-Mutators-and-profiles this page with a couple of words for this new feature